src/composite/MegaChart.js

```diff
@@ -71,8 +71,8 @@
         
             this._layout
                 .getContent("center")
-                    .data(this.data())
-                    .columns(this.columns());
+                    .columns(this.columns())
+                    .data(this.data());
```

Comment:
The order shouldn't matter - as it should not be acted on until render is called...